documentation/docs/user_guide/inputs/cable_soilparm_nml.md

```diff
@@ -1 +1,55 @@
+soilin%silt=0.08,0.33,0.17,0.2,0.06,0.25,0.15,0.7,0.33,
+soilin%ssat=0.398,0.479,0.482,0.443,0.426,0.482,0.42,0.451,0.479,
+soilin%sucs=-0.106,-0.591,-0.405,-0.348,-0.153,-0.49,-0.299,-0.356,-0.153,
+soilin%swilt=0.072,0.216,0.286,0.135,0.219,0.283,0.175,0.395,0.216,
+/
+```
+
+| Namelist variable | Type              | Available values | Default values | Description                                                                              |
+|-------------------|-------------------|------------------|----------------|------------------------------------------------------------------------------------------|
+| soilin%desc       | character(*)      |                  | uninitialised  | Description of soil types                                                                |
+| soilin%bch        | real(n_soiltypes) | >=0.0            | uninitialised  | Parameter b in Campbell equation for the pore size distribution index (1,2) \( ( - ) \). |
+| soilin%silt       | real(n_soiltypes) | >=0.0            | uninitialised  | Fraction of soil which is silt \( ( - ) \).                                              |
+| soilin%clay       | real(n_soiltypes) | >=0.0            | uninitialised  | Fraction of soil which is clay \( ( - ) \).                                              |
+| soilin%sand       | real(n_soiltypes) | >=0.0            | uninitialised  | Fraction of soil which is sand \( ( - ) \).                                              |
+| soilin%swilt      | real(n_soiltypes) | >=0.0            | uninitialised  | Volume of H2O at wilting \( (m^{3} \cdot m^{-3}) \).                                     |
+| soilin%sfc        | real(n_soiltypes) | >=0.0            | uninitialised  | Volume of H2O at field capacity \( (m^{3} \cdot m^{-3}) \).                              |
+| soilin%ssat       | real(n_soiltypes) | >=0.0            | uninitialised  | Volume of H2O at saturation \( (m^{3} \cdot m^{-3}) \).                                  |
+| soiln%hyds        | real(n_soiltypes) | >=0.0            | uninitialised  | Hydraulic conductivity at saturation \( (m^{-1}) \).                                     |
+| soilin%sucs       | real(n_soiltypes) | <=0.0            | uninitialised  | Suction at saturation \( (m) \).                                                         |
+| soilin%rhosoil    | real(n_soiltypes) | >=0.0            | uninitialised  | Soil bulk density \( (kg \cdot m^{-3}) \).                                               |
+| soilin%css        | real(n_soiltypes) | >=0.0            | uninitialised  | Soil specific heat capacity \( (J \cdot kg^{-1} \cdot K^{-1}) \).                        |
+
+
+1. Brooks, R. H., and Corey, A. T. 1964. Hydraulic Properties of Porous Media. Fort Collins: Colorado State University.
+2. Campbell, G. S. 1974. A simple method for determining unsaturated conductivity from moisture retention data. Soil Science, 117(6), 311-314. [doi:10.1097/00010694-197406000-00001
```

Comment:
```suggestion
2. Campbell, G. S. 1974. A simple method for determining unsaturated conductivity from moisture retention data. Soil Science, 117(6), 311-314. (doi:10.1097/00010694-197406000-00001)[https://doi.org/10.1097/00010694-197406000-00001]
```

Reply:
Cannot apply requested change from outdated commit. I added this to the latest commit. Thanks!